tests/test_review_context.py

```
@@ -346,195 +186,195 @@ def test_reviewer_contracts_preserve_deleted_symlink_and_optional_workspace_rule
-                freshness_status="current",
-                role="specialist",
-                attempt=1,
+            repo = Path(directory) / "repo"
+            repo.mkdir()
+            env = {**os.environ, "GIT_AUTHOR_NAME": "t", "GIT_AUTHOR_EMAIL": "t@example.com",
+                   "GIT_COMMITTER_NAME": "t", "GIT_COMMITTER_EMAIL": "t@example.com"}
+            def git(*args):
+                return subprocess.run(["git", "-C", str(repo), *args], check=True, capture_output=True, text=True, env=env).stdout.strip()
+            git("init", "-q")
+            (repo / "README.md").write_text("base\n", encoding="utf-8")
+            git("add", "README.md"); git("commit", "-q", "-m", "base without policy")
```

Ruff (0.16.3)

[error] Multiple statements on one line (semicolon)

(E702)